components/Account/Access/Campaigns/Form.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component, Fragment } from 'react'
+import { compose } from 'react-apollo'
+import isEmail from 'validator/lib/isEmail'
+
+import {
+  Button, Field, Label, InlineSpinner, Interaction, A
+} from '@project-r/styleguide'
+
+import ErrorMessage from '../../../ErrorMessage'
+import withT from '../../../../lib/withT'
+
+const { H3 } = Interaction
+
+class Form extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      email: '',
+      error: false,
+      isMutating: false,
+      hideForm: true,
+      mutationError: null,
+      isUsed: props.campaign.slots.used > 0,
+      dirty: {}
```

Comment:
Since you only have one field: no need for an object. Start with `dirty: false` or `dirty: undefined` or just leave it of of the initial state (implicit undefined).